mod-source-record-storage-server/src/test/java/org/folio/services/CreatedInstanceEventHandlingServiceImplTest.java

```diff
@@ -110,7 +111,14 @@ public void shouldSetInstanceIdToRecord(TestContext context) {
       .withContext(payloadContext);
 
     Future<Boolean> future = recordDao.saveRecord(record, TENANT_ID)
-      .compose(rec -> eventHandlingService.handle(Json.encode(dataImportEventPayload), TENANT_ID));
+      .compose(rec -> {
+        try {
+          return eventHandlingService.handle(ZIPArchiver.zip(Json.encode(dataImportEventPayload)), TENANT_ID);
+        } catch (IOException e) {
+          e.printStackTrace();
+          return null;
```

Comment:
return failed future? this way we should have an assertion error in the handler

Reply:
Changed.